app/build.gradle.kts

```diff
@@ -158,6 +158,10 @@ dependencies {
     implementation("io.ktor:ktor-client-cio:2.3.8")
     testImplementation("io.ktor:ktor-client-mock:2.3.8")
 
+    // File/Stream/IO API
+    implementation("com.squareup.okio:okio:3.8.0")
```

Comment:
I notice we already have `implementation("org.jetbrains.kotlinx:kotlinx-io-core:0.3.0")` above. It appears that both these libraries have the same purpose, so perhaps I should use `kotlinx-io-core` instead?

Reply:
We use kotlinx-io-core for a certain functionality only, for reading a ByteArray. Otherwise, it is not used. Can kotlinx-io-core do everything okio can?

Reply:
Anyway, since `kotlinx-io-core` is described experimental, I would like to avoid it at this time. If it was not experimental, I would probably slightly prefer it over okio because it is kind of official.